docs/finality-provider-operation.md

```diff
@@ -0,0 +1,732 @@
+
+```
+[INFO] Starting finality provider daemon...
+[INFO] RPC server listening on...
+```
+
+> ⚠️ **Important**: The daemon needs to run continuously. It's recommended to set 
+> up a system service (like `systemd` on Linux or `launchd` on macOS) to manage 
+> the daemon process, handle automatic restarts, and collect logs.
+
+The above will start the Finality provider RPC server at the address specified
+in `fpd.conf` under the `RpcListener` field, which has a default value
+of `127.0.0.1:12581`. You can change this value in the configuration file or
+override this value and specify a custom address using
+the `--rpc-listener` flag.
+
+All the available CLI options can be viewed using the `--help` flag. These
+options can also be set in the configuration file.
+
+### 4.5. Interaction with the EOTS Manager
+
+There are 2 pieces to a finality provider entity: the EOTS manager and the 
+finality provider instance. These components work together and are managed by 
+separate daemons(`eotsd` and `fpd`).
+
+The EOTS manager is responsible for managing the keys for finality providers and 
+handles operations such as key management, signature generation, and randomness 
+commitments. Where as the finality provider is responsible for creating and 
+registering finality providers and handling the monitoring of the Babylon chain. 
+The finality provider daemon is also responsible for coordinating various 
+operations.
+
+The interactions between the EOTS Manager and the finality provider happens 
+through RPC calls. These calls handle key operations, signature generation, 
+and randomness commitments. An easy way to think about it is the EOTS Manager 
+maintains the keys while the FP daemon coordinates any interactions with the 
+Babylon chain.
+
+The EOTS Manager is designed to handle multiple finality provider keys, operating 
+as a centralized key management system. When starting a finality provider instance, 
+you specify which EOTS key to use through the `--eots-pk` flag. This allows you 
+to run different finality provider instances using different keys from the same 
+EOTS Manager.
+
+For example, after registering a finality provider, you can start its daemon by 
+providing the EOTS public key:
+
+```shell
+fpd start --eots-pk <hex-string-of-eots-public-key>
+```
+
+## 5. Finality Provider Operations
+
+### 5.1 Create Finality Provider
+
+The `create-finality-provider` command initializes a new finality provider
+instance locally. 
+
+``` shell
+fpd create-finality-provider \ 
+  --daemon-address 127.0.0.1:12581 \ 
+  --chain-id bbn-test-5 \ 
+  --eots-pk <eots-pk-hex> \
+  --commission 0.05 \ 
+  --key-name finality-provider \ 
+  --moniker "MyFinalityProvider" \ 
+  --website "https://myfinalityprovider.com" \
+  --security-contact "security@myfinalityprovider.com" \ 
+  --details "finality provider for the Babylon network" \ 
+  --home ./fpHome
+```
+
+<!-- TODO: doesn't this also work with a JSON file, like SDK validators? -->
```

Comment:
We can after https://github.com/babylonlabs-io/finality-provider/issues/14 is addressed